package.json

```diff
@@ -4,9 +4,11 @@
   "private": true,
   "dependencies": {
     "axios": "0.24.0",
+    "bootstrap": "^5.1.3",
     "notiflix": "3.2.2",
     "query-string": "7.0.1",
     "react": "17.0.2",
+    "react-bootstrap": "^2.2.0",
```

Comment:
Why have you installed react-bootstrap & bootstrap?

Reply:
The dev branch was with problems, when I fetch this!! The app was with error cuz the react pagínate! So I just installed react pagínate and this update the bootstrap too.

Reply:
I was checked and React Paginate package.json doesn't include https://github.com/AdeleD/react-paginate/blob/master/package.json , what if remove bootstrap from this pull request and you create an issue with a screenshot of the error that you're getting?